Tools/CISupport/ews-build/steps.py

```diff
@@ -1749,6 +1749,10 @@ def run(self):
 
     @defer.inlineCallbacks
     def validate_bugzilla(self, patch_id):
+        if self.getProperty('sensitive', False) and self.getProperty('buildername', '').lower() == 'commit-queue':
+            rc = yield self.skip_build('Cannot land security changes with Commit-Queue')
```

Comment:
1) Instead of marking the build as skipped, might be better to mark it failed.
2) It's better to remove cq+ flag and leave a comment on the bug, otherwise it might be hard to discover for the engineers.
3) Better to mention the solution as well, e.g.: `please use GitHub PR instead.`

Something like:
```
message = 'XYZ'
self.setProperty('build_finish_summary', message)
self.setProperty('comment_text', message)
self.build.addStepsAfterCurrentStep([LeaveComment(), SetCommitQueueMinusFlagOnPatch()])
```

Note that SetCommitQueueMinusFlagOnPatch will mark the build as failed (if `build_finish_summary` property is set).